stories/module-analytics-setup.stories.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * Analytics Setup stories.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { removeAllFilters, addFilter } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import SetupWrapper from '../assets/js/components/setup/setup-wrapper';
+import { SetupMain as AnalyticsSetup } from '../assets/js/modules/analytics/setup/index';
+import { fillFilterWithComponent } from '../assets/js/util';
+import * as fixtures from '../assets/js/modules/analytics/datastore/__fixtures__';
+
+import { STORE_NAME } from '../assets/js/modules/analytics/datastore';
+import { WithTestRegistry } from '../tests/js/utils';
+import { ACCOUNT_CREATE } from '../assets/js/modules/analytics/datastore/constants';
+
+function filterAnalyticsSetup() {
+	global.googlesitekit.setup.moduleToSetup = 'analytics';
+
+	removeAllFilters( 'googlesitekit.ModuleSetup-analytics' );
+	addFilter(
+		'googlesitekit.ModuleSetup-analytics',
+		'googlesitekit.AnalyticsModuleSetup',
+		fillFilterWithComponent( AnalyticsSetup )
+	);
+}
+
+function Setup( props ) {
+	return (
+		<WithTestRegistry { ...props }>
```

Comment:
We should probably have a generic `render` function for storybook tests to use, as this is replicated a lot throughout the stories. The Jest tests do something similar—we might even be able to share between the tests?

Reply:
I don't think the RTL `render` can be used like that, if that's what you're suggesting? We could definitely abstract this a little more to remove the repetitive parts in each.